src/lib/zod2form.ts

```diff
@@ -0,0 +1,40 @@
+import * as z from "zod";
+import _ from "lodash";
+
+/*
```

Comment:
this comment is not in line with the code below. so either the code or the comment are wrong.

the code generate:

{ "table": [ { "name": ["error 1"] } ] }